blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/RollupRunnable.java

```diff
@@ -165,6 +165,7 @@ public void run() {
                     singleRollupReadContext.getRange().toString(),
                     srcGran.name(),
                     e});
+            executionContext.markUnsuccessful(e);
```

Comment:
The above addition of the ```executionContext.markUnsuccessfull()``` had bad effects, lots of tests are failing. I'm removing it.